### PR TITLE
Change internal font size from from 15 to 10

### PR DESCRIPTION
**Pull request by JorikSchellekens**
We feared that using non integer rem values would cause strange rounding artefacts. We had done some testing which showed that largely this wasn't a worry. However, I've run into the a strange bug on chrome, mac OS (not safari or firefox) which uses a different rounding for vertical and horizontal values. It effects the third decimal significantly. It's usually unoticable but on small circles it's glaring. There was a work around for the toggle switches but now the new radio buttons were looking very disproportionate.

This pr changes our internal base font to 10, making the value of all our $font-Xpx variables a single decimal place long which don't suffer from the rounding problem.

The artefact doesn't show up when you change the base with the slider using the new single decimal place rem values. So, Chrome must be rounding early.

Before:
image

After:
image

### CODE REVIEW

File: src/settings/Settings.js [hunk 1]
```diff
@@ -180,7 +180,7 @@ export const SETTINGS = {
     "fontSize": {
```
Comment (t3chguy) [inline src/settings/Settings.js:180]: so given that we aren't changing the settings key, what effect will this have if someone changed their font already?

Comment (JorikSchellekens) [reply src/settings/Settings.js:180]: We are changing the settings key. That's why I took it off of the review queue

File: src/settings/watchers/FontWatcher.ts [hunk 2]
```diff
@@ -45,11 +47,11 @@ export class FontWatcher implements IWatcher {
     };
 
     private setRootFontSize = (size) => {
-        // Externally we tell the user the font is size 15. Internally we use 10.
-        const fontSize = Math.max(Math.min(FontWatcher.MAX_SIZE, size), FontWatcher.MIN_SIZE) - 5;
+        console.log({size})
```
Comment (t3chguy) [inline src/settings/watchers/FontWatcher.ts:50]: debug

**Comment by t3chguy** [comment]
assuming you get it passing tests

**Comment by t3chguy** [comment]
Looks like either your base branch is out of date or you have a branch of the same name in riot-web which is clashing

**Comment by JorikSchellekens** [comment]
Yeah, that's something travis has warned me about. I still had a bunch of open branches on my current fork but I'll be moving to matrix-orgs one monday

**Comment by JorikSchellekens** [comment]
Closing this in favour of https://github.com/matrix-org/matrix-react-sdk/pull/4725

**Pull request closed without merging**